### PR TITLE
Update Project-Contribution.md

### PR DESCRIPTION
**Pull request by aitana16**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Project-Contribution.md [hunk 1]
```diff
@@ -26,8 +26,15 @@ assignees: 'maoo, aitana16'
 Below is the list of tasks that FINOS Team and the contribution author go through in order to complete the FINOS contribution process.
 **Please do not edit these contents at contribution time!**
 
-## Identify project meta
-- [ ] Project Title
+## FINOS Contrib POC
+- [ ] Identify and Assign FINOS Contrib POC
+
+## Identify project meta (Lead: FINOS Contrib POC)
```
Comment (maoo) [inline .github/ISSUE_TEMPLATE/Project-Contribution.md:32]: I'd suggest adding Marketing as support, as they're going to work on logo, landscape-related questions and naming.

File: .github/ISSUE_TEMPLATE/Project-Contribution.md [hunk 2]
```diff
@@ -49,28 +56,28 @@ Below is the list of tasks that FINOS Team and the contribution author go throug
...
 
-## Code validation (only if code is contributed)
+## Code validation (only if code is contributed) (Lead: FINOS Infra)
 - [ ] The codebase doesn’t have HIGH or CRITICAL CVEs across direct and transitive libraries
 - [ ] The codebase doesn’t have any unfriendly licenses across direct and transitive libraries
 - [ ] (optional - if a build system is provided) The build process runs successfully
   - [ ] [finos-admin](http://github.com/finos-admin) is Admin of the GitHub repository to transfer
 - [ ] The codebase doesn’t include any patent or copyright that conflicts with FINOS Governance and bylaws (to be validated with FINOS Legal team)
 - [ ] The codebase complies with FINOS Project Blueprint
+    - [ ] Create new CONTRIBUTING.template.md
```
Comment (maoo) [inline .github/ISSUE_TEMPLATE/Project-Contribution.md:66]: Probably you meant `Create new CONTRIBUTING.md` ? Well, this should come with the project blueprint, so there's no reason to add this item here.

Also, I expect there will be a `GOVERNANCE.md` too, right @mindthegab @copiesofcopies  ? Is there any special activity to be done when a project repo is created, other than apply replacement for placeholders, as already documented on https://odp.finos.org/docs/project-collaboration/#finos-project-blueprint ?

**Review by maoo — CHANGES_REQUESTED** [review]
Great work! I added some suggestions

File: .github/ISSUE_TEMPLATE/Project-Contribution.md (hunk 1 above)
Comment (aitana16) [reply .github/ISSUE_TEMPLATE/Project-Contribution.md]: Good point, added. Thanks!

**Pull request merged**